src/js/plugin/external/PluginActions.js

```diff
@@ -0,0 +1,7 @@
+import Util from "../../helpers/Util";
+
+const PluginActions = Util.objectCreateWithAdder("PLUGIN_ACTION_");
+
+["DIALOG_ALERT"].forEach(PluginActions.add);
```

Comment:
Yes, I think this could be a better - more usual - way.
I will take car about this.